apps/examples/libtuv/Makefile

```diff
@@ -134,10 +134,10 @@ install:
 endif
 
 ifeq ($(CONFIG_EXAMPLES_LIBTUV),y)
```

Comment:
Even if builtin functionality is disabled, this tab will be called. Let's add CONFIG_BUITIN_APPS